File: server/models/milestone.js
const { DataTypes } = require('sequelize');

module.exports = sequelize => {
  sequelize.define('milestone', {
    title: {
      allowNull: false,
      type: DataTypes.STRING,
    },
    description: {
      type: DataTypes.STRING,
    },
    dueDate: {
      type: DataTypes.DATE,
    },
    isOpen: {
      allowNull: false,
      type: DataTypes.BOOLEAN,
      defaultValue: true,
    },
  });
};

File: client/iOS/IssueTracker/IssueTracker/Extension/Model/Milestone.swift
//
//  Milestone.swift
//  IssueTracker
//
//  Created by A on 2020/11/02.
//

import Foundation

struct Milestone: Codable, Hashable {
    let id: Int
    let title: String
    let description: String?
    let isOpen: Bool?
    let dueDate: String?
    let createdAt: String
    let updatedAt: String
}
